packages/radio-box-group/src/RadioBoxGroup.spec.tsx

```diff
@@ -9,7 +9,7 @@ afterAll(cleanup);
 const className = 'test-radio-box-class';
 
 const { container } = render(
-  <RadioBox value="radio-1" className={className} checked={true}>
+  <RadioBox value="radio-1" className={className} checked readOnly>
```

Comment:
I'm curious why we're adding the read-only attribute here. Seems unrelated to the PR.

Reply:
This was to silence:
>Warning: Failed form propType: You provided a checked prop to a form field without an onChange handler. This will render a read-only field. If the field should be mutable use defaultChecked. Otherwise, set either onChange or readOnly

Reply:
Ahh nice 👍